src/ua/skrypnyk/hw12/ArrayList.java

```diff
@@ -0,0 +1,50 @@
+package src.ua.skrypnyk.hw12;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class ArrayList<T> {
```

Comment:
Here is result of your code:
[null, null, null, null, null, null, 1]
Please, use index variable to find out the right place for adding element